etc/mail-annoucement.txt

```diff
@@ -0,0 +1,27 @@
+Hello HTTP working group,
+
+in the past months we have been working on a draft for resumable file uploads using HTTP and want to present our work to you. To briefly recap, resumable uploads allow resuming an upload where it has been stopped. For example, after the connection has been interrupted. A 100 MB upload, which was interrupted after transmitting 70 MB, only needs to transmit the remaining 30 MB.
+
+There is support in HTTP for resumable downloads using the Range header, but no support for resumable uploads so far. There have been attempts to standardize resumable uploads, such as [1], but none completed this journey yet to our knowledge.
```

Comment:
I would reorder these a bit to say something along the lines of 

"We are all familiar with connectivity disruptions affecting our Internet activities. One example, is when a large file download is interrupted; say a 100 MB file download encounters a network loss after the client receives 70 MB. Fortunately, resumable HTTP downloads using range requests are a widely deployed standard feature that allows clients to fetch the remaining 30 MB only, saving time and resources for both endpoints. However, in the opposite direction, there is not a standard convention for resuming HTTP uploads. 

Across the HTTP ecosystem there are several different approaches to providing resumable uploads. We are aware of at least one attempt to try and standardize an approach [1], but to our knowledge none have succeeded in being adopted and driven to conclusion.